packages/foreman/foreman-installer/foreman-installer.spec

```
@@ -139,6 +139,9 @@ done
 %{_sbindir}/foreman-proxy-certs-generate
 
 %changelog
+* Thu Jan 09 2020 b'Patrick Creech <pcreech@redhat.com>' - b'1:1.24.1-1'
```

looks like you're hitting the same issue i noticed when bumping develop to 2.0 - these `b'` tags seem to indicate some python string encoding problem in obal and should be dropped like so:
```suggestion
* Thu Jan 09 2020 Patrick Creech <pcreech@redhat.com> - 1:1.24.1-1
```